index.html

```diff
@@ -227,5 +234,28 @@ <h1>Skills</h1>
             </div>
         </div>
     </section>
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form>
+                <input type="text" maxlength="30" placeholder="Full name" required>
+                <input type="email" placeholder="Email adress" required>
+                <textarea maxlength="500" placeholder="Enter text here" required></textarea>
+                <button name="submit">Get in touch</button>
+                <ul>
+                    <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href=""><i class="fa-brands fa-linkedin-in"></i></a></li>
+                    <li><a href=""><i class="fa-brands fa-github"></i></a></li>
+                    <li><a href=""><i class="fa-solid fa-envelope"></i></a></li>
+                </ul>
```

Comment:
- [ ] Kindly use the same social media icons as mentioned in the Figma design check below :point_down: 

Your Design:
![Screenshot from 2023-02-21 21-01-12](https://user-images.githubusercontent.com/45226848/220401007-3eabce1a-0c75-4e29-98f7-892fdebd2b37.png)

Figma Design:
![Screenshot from 2023-02-21 21-01-04](https://user-images.githubusercontent.com/45226848/220401068-c4035693-8da3-45ec-9575-252816105e88.png)